### DISCUSSION TITLE
Wechselrichter zeigen ! und x -> dann falsche/ keine Werte

### DISCUSSION THREAD

**Discussion by gkellershs** [tbnobody/OpenDTU#917] [General]
Ich habe gestern die Module zusammengebaut und konfiguriert. Die Wechselrichter (2x HM-1500) werden erkannt und Werte werden geliefert. 
Allerdings bekomme ich sehr häufig Fehler, welche ich nicht deuten kann. Dabei bekommen in der Live Ansicht die Wechselrichter ein ! oder x im Symbol und es werden scheinbar auch keine oder sogar falsche Werte übertragen.

Was genau bedeutet ein ! und was ein x und wie kann ich nach einer Lösung suchen?

Vielen Dank für Infos und Hilfe, Georg

![image](https://github.com/tbnobody/OpenDTU/assets/124203296/8a0b4494-d620-41e1-9086-f2461b34cadc)
![image](https://github.com/tbnobody/OpenDTU/assets/124203296/49fe16ea-451e-4eac-991b-6d44dd349da1)
![image](https://github.com/tbnobody/OpenDTU/assets/124203296/c2c77b69-8869-40e6-b1d1-d7271cb92889)

**Comment by gkellershs** [comment]
Nachtrag zu meinem Post: 
Ich habe parallel eine "original Hoymiles DTU" am laufen, diese habe ich versuchsweise ausgemacht und es scheint, als ob das Problem damit gelöst ist. Ist das ein bekanntes Problem?

**Comment by tbnobody** [comment]
Ja, das ist bekannt und lässt sich nicht beheben.
